tests/SampleApps/python/http-server-py/myapp.py

```diff
@@ -0,0 +1,8 @@
+
+def app(environ, start_response):
+    data = b"Hello Gunicorn!\n"
+    start_response("200 OK", [
+        ("Content-Type", "text/plain"),
+        ("Content-Type", str(len(data)))
```

Comment:
content-length?

Reply:
Good catch, this should be `Content-Length`